samples/client/petstore/typescript-axios/tests/default/tsconfig.json

```diff
@@ -1,20 +1,13 @@
 {
-    "compilerOptions": {
-        "moduleResolution": "node",
-        "module": "ES5",
-        "target": "ES5",
-        "noImplicitAny": true,
-        "sourceMap": false,
-        "outDir": "dist",
-        "types": [
-            "mocha"
-        ],
-        "lib": [
-            "es6",
-            "dom"
-        ]
-    },
-    "exclude": [
-        "node_modules"
-    ]
+  "compilerOptions": {
```

Comment:
It looks like this file was formatted, can you please re-generate it and commit it as it is?